src/webchat-ui/components/plugins/input/base/BaseInput.tsx

```diff
@@ -71,10 +72,23 @@ const Button = styled.button(({ theme }) => ({
 	},
 }));
 
-const AttachFileButton = styled(Button)(({ theme }) => ({}));
+const AttachFileButton = styled(Button)(({ theme }) => ({
```

Comment:
The styles for AttachFileButton, SpeechButton, and SubmitButton are identical. Could you create a shared constant to avoid redundancy?

Reply:
good idea. fixed